static/js/components/UserManagement.jsx

```diff
@@ -271,6 +368,7 @@ const UserManagement = () => {
     const results = await Promise.all(promises);
     if (results.every((result) => result.status === "success")) {
       dispatch(showNotification("User(s) successfully invited."));
+      dispatch(invitationsActions.fetchInvitations());
```

Comment:
This fetch is just to have the new invite show up on the management page, right? I think it would be better to do that through a webhook instead for consistency with the rest of SP

Reply:
I don't think what you're proposing is the right pattern here. Note that the invitations will never be fetched for most users (only fetched when a system admin user loads the user management page, not in hydrate), so for most users that branch of the state will always be null, as opposed to changes to sources, e.g., which should use the websocket push. Most users don't even have access to the endpoint to fetch invitations.

Reply:
Ah okay yeah, you're right let's keep this as is.